packages/nocodb/src/db/BaseModelSqlv2.ts

```diff
@@ -1621,7 +1621,16 @@ class BaseModelSqlv2 {
 
         tQb.select(jsonBuildObject);
 
-        selectors.push(this.dbDriver.raw(`(??) as ??`, [tQb, `${f.alias}`]));
+        if (this.dbDriver.client.config.client === 'mysql2') {
+          selectors.push(
+            this.dbDriver.raw(`JSON_UNQUOTE((??)) as ??`, [
+              jsonBuildObject,
+              `${f.alias}`,
```

Comment:
`'${f.alias}' `=> `f.alias`

Reply:
If the f.alias is number or some other special character, it may cause issue. That's why kept in ``